content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,54 @@
+# Add Dependencies or Outputs to a Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that some of its data requirements are missing from `dvc.yaml`. Namely, one of:
+
+- Files or directories in the <abbr>workspace</abbr> that are dependencies of
+  the stage are missing from `deps`.
+- Output files or directories that the stage creates, which are already in the
+  workspace are missing from `outs`.
+
+Follow the steps below to add existing files/directories as
+<abbr>dependencies</abbr> or <abbr>outputs</abbr> to a stage without executing
+it again, which can be expensive/time-consuming, and is unnecessary.
```

Comment:
Ok so this can be simplified quite a bit probably, as mentioned in https://github.com/iterative/dvc.org/pull/1914#pullrequestreview-534022299. E.g.:

> all we need to describe in this section is open dvc.yaml - edit or execute dvc run again. If it happened that files are already in place use --no-exec + dvc commit

> much simpler scenario and recommendation first - Open your file, add it manually, save file. Run repro or commit in some cases.

Reply:
Simplified this into two small paragraphs.

Reply:
Very good, thanks.